scalameta/parsers/shared/src/main/scala/scala/meta/internal/parsers/ScalametaParser.scala

```diff
@@ -820,16 +821,31 @@ class ScalametaParser(input: Input)(implicit dialect: Dialect) { parser =>
     val result = body
     var endTokenPos = end.endTokenPos
     if (endTokenPos < startTokenPos) endTokenPos = startTokenPos - 1
-    val pos = TokenStreamPosition(startTokenPos, endTokenPos + 1)
+    val pos = trimmedPos(startTokenPos, endTokenPos)
     result.withOrigin(Origin.Parsed(input, dialect, pos)).asInstanceOf[T]
   }
 
+  def trimmedPos(startTokenPos: Int, endTokenPos: Int): TokenStreamPosition = {
+    def skipTrivia(range: Range): Int = {
+      range
+        .dropWhile(i => scannerTokens(i).is[Trivia])
```

Comment:
is `find(!...)` available on `Range`?

Reply:
`dropWhile` is specifically implemented for `Range` and should work slightly faster than generic `find`